frontend/src/less/variables.less

```diff
@@ -3,3 +3,4 @@
 @color-default: #0e2d3b;
 @color-black: #222;
 @brand-primary: @color-primary;
+@width-searchbox: 563px;
```

Comment:
@amcatko please add newline to the end of file, and ideally configure your editor to do it automatically